node_modules/oae-tenants/lib/api.js

```diff
@@ -203,6 +281,60 @@ var _cacheTenants = function(callback) {
             tenantsByHost[tenant.host] = tenant;
         });
 
+        // Cache the sorted list
+        sortedTenants = _.sortBy(tenants, 'alias');
+
+        // Build the search index for all tenants
+        tenantSearchIndex = new TenantIndex(tenants);
+
+        // Indicate that all tenants have been cached
+        TenantsAPI.emit('cached');
+
+        return callback(null, tenants);
+    });
+};
+
+/**
+ * Fetch the tenant and update it's entry in the cache
+ *
+ * @param  {String}         tenantAlias         The alias of the tenant to be re-cached
+ * @param  {Function}       [callback]          Standard callback function
+ * @param  {Object}         [callback.err]      Error object containing the error code and message
+ * @api private
+ */
+var _updateCachedTenant = function(tenantAlias, callback) {
+    callback = callback || function(err) {
+        if (err) {
+            log().error({'err': err}, 'Failed to re-cache the tenants');
+        }
+    };
+
+    // Get the available tenants
+    Cassandra.runQuery('SELECT * FROM "Tenant" WHERE "alias" = ?', [tenantAlias], function(err, rows) {
+        if (err) {
+            return callback(err);
+        }
+
+        _.each(rows, function(row) {
```

Comment:
There should only be 1 row as the `alias` field is the PK for this column family. I think we can get rid of this `_.each` here